composer/datasets/in_context_learning_evaluation.py

```diff
@@ -33,17 +31,43 @@
 ]
 
 
-def strip_data(samples):
-    return [{k: v.strip() if isinstance(v, str) else v for k, v in entry.items()} for entry in samples]
+def _check_if_huggingface_uri(uri: str) -> bool:
+    """
+    Takes a dataset uri and checks if it's a HuggingFace dataset uri.
+    Returns False if a backend uri is present (ie 's3://', 'oci://') or if the uri is a local file.
+    Args:
+        uri (str): uri as a string
+
+    Returns:
+        bool: result of parsing uri as a HF uri
+    """
+    backend, _, path = parse_uri(uri)
+    # If there's any backend, it's a cloud OCI and not HF
+    if backend == '':
```

Comment:
I'm feeling a bit iffy about relying on file extensions. Given that they aren't actually otherwise required anywhere else. Could we instead use a special `hf://` prefix?

Reply:
Okay, can do!